Tools/CISupport/build-webkit-org/config.json

```diff
@@ -292,6 +292,11 @@
                       "platform": "mac-bigsur", "configuration": "release", "architectures": ["x86_64", "arm64"],
                       "workernames": ["bot184"]
                     },
+                    { "name": "Apple-BigSur-XCBuild-Release-Build", "factory": "BuildFactory", "builddir": "bigsur-release",
```

Comment:
Looks like EWS caught a bug for you:
```
Configuration Errors:
  duplicate builder builddir 'bigsur-release'
```

Reply:
Now I have to wonder why `python3 runUnittests.py build-webkit-org` told me all tests passed!

Reply:
It was `buildbot checkconfig` that actually flagged the failure, looking at the logs.